src/filters/defaults/alpha/AlphaFilter.ts

```diff
@@ -37,8 +37,7 @@ export interface AlphaFilterOptions
  */
 export class AlphaFilter extends Filter
 {
-    public static readonly defaultOptions: AlphaFilterOptions & Partial<FilterOptions> = {
-        ...Filter.defaultOptions,
```

Comment:
why was this removed?
shouldn't you be able to set default settings per filter?